src/main/java/edu/kis/vh/nursery/FIFORhymer.java

```diff
@@ -1,19 +1,35 @@
 package edu.kis.vh.nursery;
 
+import edu.kis.vh.nursery.memory.IntArrayStack;
+import edu.kis.vh.nursery.memory.IntMemory;
+
 public class FIFORhymer extends DefaultCountingOutRhymer {
 
-    private final DefaultCountingOutRhymer temp = new DefaultCountingOutRhymer();
+    // IntArrayStack is the best option here, looking at the type of operations.
+    // We are performing a lot of accessing, no inserting in the middle.
+    // Inserting in the middle or at the beginning would be LinkedList's edge.
+    // With an array, we sacrifice dynamic-size over memory taken (Nodes take up space).
```

Comment:
I understand explanation but the unlimited size is more important than efficiency.